src/SumoCoders/FrameworkCoreBundle/Tests/Controller/DatePickerControllerTest.php

```diff
@@ -97,22 +65,14 @@ public function testIfSingleTextWithDatePickerAndOnlyDatesInTheFutureRenderedCor
         $startDate = new \DateTime('last monday');
 
         $element = $crawler->filter('input#form_date_example6');
```

Comment:
don't forget to adjust this id when shifting the ids in the code

Reply:
fixed 89a0a2e